### PR TITLE
Add pacmd set-sink wrapper module

### PR DESCRIPTION
**Pull request by c4pQ**

### CODE REVIEW

File: pacmd/__init__.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+"""Sets default audio sink
+
+ Synopsis: <trigger> <query>"""
+
+import subprocess
+from albert import *
+
+__title__ = "Pulseaudio cmd"
```
Comment (ManuelSchneid3r) [inline pacmd/__init__.py:10]: no need for the "cmd"

File: pacmd/__init__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+__version__ = "0.1.0"
+__triggers__ = "pacmd "
```
Comment (ManuelSchneid3r) [inline pacmd/__init__.py:12]: i'd tend to use just pa

File: pacmd/__init__.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+__authors__ = "c4pQ"
+__exec_deps__ = ['pacmd']
+
+iconPath = iconLookup('audio-headphones')
```
Comment (ManuelSchneid3r) [inline pacmd/__init__.py:16]: see https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/blob/master/pulseaudio-text.svg i dunno if we can expect that 'pulseaudio-text' is on every system but id rather use this one

**Review by ManuelSchneid3r — CHANGES_REQUESTED** [review]

**Comment by ManuelSchneid3r** [comment]
Neat. But i dont see the need to be to narrow. This is basically a PA wrapper. I dont expect you to wrap the entire api. This could serve as a starting point for others who need futher functionality.

File: pacmd/__init__.py (hunk 1 above)
Comment (c4pQ) [reply pacmd/__init__.py]: removed

File: pacmd/__init__.py (hunk 2 above)
Comment (c4pQ) [reply pacmd/__init__.py]: makes sense

**Comment by c4pQ** [comment]
Added requested changes

**Comment by ManuelSchneid3r** [comment]
0.18 is out. please check the new api https://github.com/albertlauncher/plugins/blob/master/python/README.md

**Comment by ManuelSchneid3r** [comment]
@c4pQ are you still interested in getting this plugin upstream?

**Comment by ManuelSchneid3r** [comment]
I'll close this one due to the conflicts. Note that this has no further implication for me but to be able to work with the PR section in this repo efficiently. It is quite exhausting to keep track of the few active PRs between dozens of stale PRs. Please reopen or open a new PR if you are ready to merge.

**Pull request closed without merging**